client/src/public/ts/index.ts

```
@@ -46,21 +51,44 @@ serifFont.load();
+  });
+  await initializeTextures();
+  initializeStageItems();
+  setContainerItemProperties();
+  document.getElementById("canvas-container")?.appendChild(app.canvas);
+  // app.renderer.view.style.position = "absolute";
+  // app.renderer.view.style.display = "block";
+  app.stage.addChild(playerContainer);
+  app.ticker.add((deltaTime) => {
+    render(app.ticker.elapsedMS);
+  });
+  for (let item in stageItems.sprites) {
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'item' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)